tools/cldr-code/src/main/java/org/unicode/cldr/json/Ldml2JsonConverter.java

```diff
@@ -1232,7 +1247,13 @@ public void writeReadme(String outputDir, String packageName) throws IOException
         }
         try (PrintWriter outf =
                 FileUtilities.openUTF8Writer(outputDir + "/" + packageName, "LICENSE"); ) {
-            FileCopier.copy(CldrUtility.getUTF8Data(CldrUtility.LICENSE), outf);
+            if (licenseFile.isPresent()) {
+                try (BufferedReader br = FileUtilities.openUTF8Reader("", licenseFile.get()); ) {
+                    FileCopier.copy(br, outf);
+                }
+            } else {
+                FileCopier.copy(CldrUtility.getUTF8Data("unicode-license.txt"), outf);
```

Comment:
I didn't get to actually run this before.. but now that I am, it's failing.  Why was the name `unicode-license.txt` used here? That's actually _not_ in the build system, and hasn't been for some time. It should have kept the name LICENSE.  I'll reopen this and make a PR.